### DISCUSSION TITLE
how do i add the song to freeplay

### DISCUSSION THREAD

**Discussion by stripes10122** [JordanSantiagoYT/FNF-JS-Engine#667] [General]
i charted it and did the json, and its not in freeplay, help me

**Comment by JordanSantiagoYT** [comment]
you have to make a weeks folder, and inside it put a week.json file with the name of the song. the name of the week json can be anything but it can't match with another week json's name

**Reply by stripes10122** [reply]
i did it, iw as stupid, now i got aa  ogn with 50k notes that is 1 minite long, now i will do a 1 million note short song for craps and giggles
